streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -1521,43 +1529,35 @@ public synchronized boolean close(final CloseOptions options) throws IllegalArgu
         if (timeoutMs < 0) {
             throw new IllegalArgumentException("Timeout can't be negative.");
         }
+        log.debug("Stopping Streams client with timeoutMillis = {} ms.", timeoutMs);
+        return close(timeoutMs, options.leaveGroup);
```

Comment:
Seems we are missing a non-null check for `options`? (Unrelated to this PR, but we should just fix it on the side -- should be the first line of code in this method.)

Reply:
+1.